api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java

```diff
@@ -16,7 +17,10 @@
  */
 @ThreadSafe
 final class DefaultOpenTelemetry implements OpenTelemetry {
-  private static final OpenTelemetry NO_OP = new DefaultOpenTelemetry(ContextPropagators.noop());
+  private static final OpenTelemetry NO_OP =
+      IncubatingUtil.incubatingApiIfAvailable(
+          new DefaultOpenTelemetry(ContextPropagators.noop()),
+          "io.opentelemetry.api.incubator.ExtendedDefaultOpenTelemetry");
```

Comment:
> There is no class with this FQCN. 

good point

> and we can get rid of `AbstractOpenTelemetry` since only one interface exists.

this class doesn't exist

Reply:
The test is abstract because there are 2 variants - with and without incubating. Should I change the name?